src/main/java/org/jenkinsci/plugins/mesos/listener/MesosRunListener.java

```diff
@@ -73,4 +82,11 @@ private final Node getCurrentNode() {
     return null;
   }
 
+  private boolean isBuildRunningInsideContainer(EnvVars environment) {
+    if(environment.get(MESOS_CONTAINER_NAME) != null) {
```

Comment:
In Mesos, a container can be run using Mesos Containerizer or Docker containerizer. AFAICT, this env variable is only set when using Docker containerizer. So we should rename this method name to `isBuildRunningwithDockerContainerizer` but if we take my above suggestion, we can kill this method altogether.